### PR TITLE
DSD-1054: Footer

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-1073](https://jira.nypl.org/browse/DSD-1073)

## This PR does the following:

- The React `Footer` component passed QA and we can merge it into the repo. I set it to `1.1.0` and it can wait until 8/14 (next release date if not 8/4).
- Everything in this PR has already been peer reviewed. This PR really only updates the version and points to `development`.
- The static version isn't part of the React component implementation so that can come later.

## How has this been tested?

Locally in Storybook and Turbine.

## Accessibility concerns or updates

- There is accessibility documentation in the `Footer` Storybook page, but this is based on the existing `Footer`. If we ever update the `Footer`, that accessibility information should then be updated.

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] Review the Vercel preview deployment once it is ready.

### CODE REVIEW

File: src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+interface FooterProps {
+  /** Additional class name to add. */
+  className?: string;
+  /** ID that other components can cross reference for accessibility purposes */
+  id?: string;
+}
+
+/**
+ * NYPL `Footer` component with necessary links to NYPL.org pages and NYPL
```
Comment (bigfishdesign13) [inline src/components/Footer/Footer.tsx]: To keep in line with the rest of the documentation, can this be written as a complete sentence?

File: src/components/Footer/Footer.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+const MyApp => (
+  <TemplateAppContainer
+    header={...}
+    contentPrimary={...}
+    contentSidebar={...}
+    sidebar="left"
+    footer={<Footer />}
+    renderFooterElement={false}
+  >
+    <Footer />
```
Comment (bigfishdesign13) [inline src/components/Footer/Footer.stories.mdx]: The `footer` prop is being used on `TemplateAppContainer`, so this is not needed.

**Review by bigfishdesign13 — CHANGES_REQUESTED** [review]
One of the example code blocks needs an adjustment.

**Comment by EdwinGuzman** [comment]
Updated.

**Review by bigfishdesign13 — APPROVED** [review]
Great.

**Pull request merged**